internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec_test.go

```diff
@@ -1173,6 +1173,17 @@ func Test_verifyOutputURL(t *testing.T) {
 			},
 			want: false,
 		},
+		{
+			name: "With Loki without URL and NOT default output",
```

Comment:
Nit. Update name.  It does not matter if it is the default output or not.  It will always need a URL.  Our migration of the default output should craft the URL so it should have one by the time it gets to the validation.

Reply:
otherwise lgtm